supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -410,9 +410,11 @@
 ==== image:images/yes.png[yes] SSL/TLS (noun)
 *Description*: SSL/TLS refers to the Secure Socket Layer protocol (SSL) and its successor, the Transport Layer Security protocol (TLS). As both of these protocols are frequently called "SSL", always use "SSL/TLS" to avoid confusion.
```

Comment:
It is contradictory to say "always use SSL/TLS" and then advise to use SSL/TLS only in high-level documentation entries. Can these recommendations be combined? For example, "As both of these protocols are frequently called "SSL", use "SSL/TLS" in high level documentation entries..."